osu.Game/Screens/Multi/Header.cs

```diff
@@ -40,39 +38,12 @@ public Header(ScreenStack stack)
                     Padding = new MarginPadding { Horizontal = SearchableListOverlay.WIDTH_PADDING + OsuScreen.HORIZONTAL_OVERFLOW_PADDING },
                     Children = new Drawable[]
                     {
-                        new FillFlowContainer
+                        title = new MultiHeaderTitle
                         {
                             Anchor = Anchor.CentreLeft,
                             Origin = Anchor.BottomLeft,
-                            Position = new Vector2(-35f, 5f),
-                            AutoSizeAxes = Axes.Both,
-                            Direction = FillDirection.Horizontal,
-                            Spacing = new Vector2(10f, 0f),
-                            Children = new Drawable[]
-                            {
-                                new SpriteIcon
-                                {
-                                    Size = new Vector2(25),
-                                    Icon = FontAwesome.fa_osu_multi,
-                                },
-                                new FillFlowContainer
-                                {
-                                    AutoSizeAxes = Axes.Both,
-                                    Direction = FillDirection.Horizontal,
-                                    Children = new[]
-                                    {
-                                        new OsuSpriteText
-                                        {
-                                            Text = "multiplayer ",
-                                            Font = OsuFont.GetFont(size: 25)
-                                        },
-                                        screenType = new OsuSpriteText
-                                        {
-                                            Font = OsuFont.GetFont(weight: FontWeight.Light, size: 25)
-                                        },
-                                    },
-                                },
-                            },
+                            Icon = FontAwesome.fa_osu_multi,
```

Comment:
can move icon and title into the class itself. would consider going as far as making them protected to enforce this going forward.